### PR TITLE
Updated README and tap runner

### PR DESCRIPTION
**Pull request by delvedor**
Hi all,
This is just a small pr with even small changes that are hard to land in other places :P

- Use all the cores available for running the tests
- cagataycali is no longer a contributor
- letzdoit is no longer a sponsor

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,7 +191,6 @@ Team members are listed in alphabetical order.
 
 This project is kindly sponsored by:
 - [nearForm](http://nearform.com)
-- [LetzDoIt](http://www.letzdoitapp.com/)
```
[the remark this answers is not in the document]
Comment (delvedor) [reply README.md]: Maybe a `Past Sponsors` section?

File: README.md [hunk 2]
```diff
@@ -180,7 +180,6 @@ Team members are listed in alphabetical order.
 ### Collaborators
 
 * [__Tommaso Allevi__](https://github.com/allevo), <https://twitter.com/allevitommaso>, <https://www.npmjs.com/~allevo>
-* [__Çağatay Çalı__](https://github.com/cagataycali), <https://twitter.com/cagataycali>, <https://www.npmjs.com/~cagataycali>
```
[the remark this answers is not in the document]
Comment (delvedor) [reply README.md]: `Past Collaborators`?

**Comment by mcollina** [comment]
sure

Il giorno mer 19 set 2018 alle ore 11:27 Tomas Della Vedova <
notifications@github.com> ha scritto:

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mcollina) [reply README.md]: +1

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcollina) [reply README.md]: +1

**Pull request merged**